README.md

```diff
@@ -12,6 +12,10 @@ Before executing npm commands set your Node to `lts/hydrogen`, or `v18.16.1`.
 
 Please read [Code of conduct](./code-of-conduct.md) before contributing.
 
+## Configure your IDE plugins
+
+For the same code writing style, read this [configuration info](./ide-plugins.md).
```

Comment:
Let's just say, that you should install vscode prettier plugin. We also use tailwind sorter plugin. To format your file press such key combo. No need in additional file

Reply:
How will we edit names of **_plugin_**/`package`/**file** in markdown?

Reply:
Do you mean formatting?
If yes, then it's up to you. It's not critical at all